docs/intro/install.rst

```diff
@@ -118,6 +118,27 @@ Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
 
   conda install -c conda-forge scrapy
 
+To install Scrapy on Windows using ``pip`` :-
+
+1. Install latest version of `Python`_ (Recommended to tick [Add Python to PATH] checkbox during installation)
+
+2. Install or upgrade pip::
+
+    python -m pip install --upgrade pip
+
+3. Create Venv (Virtual Environment)::
```

Comment:
I think this step and the last one about installing Scrapy itself could be removed from these steps and, instead, we could add a paragraph after the steps that says “install Scrapy using pip” which links back up to the “Installing Scrapy” section, which already suggests creating a virtual environment.

In “Installing Scrapy”, maybe we should move the paragraph about virtual environments 1 paragraph up, so that it comes right after the `pip` command, and before the paragraph that links to these Windows-specific steps.